api/fixtures/example.go

```diff
@@ -35,6 +35,17 @@ type ExampleOptions struct {
 	AWSCredentials   []byte
 	SSHKey           []byte
 	NodePoolReplicas int
+
+	// AWS-specific options
```

Comment:
Nit: what about extracting the AWS-isms (including AWSCredentials) into its own struct?